### DISCUSSION TITLE
如何实现动态友链，按照文档写的并没有生效，network也没有看到有该接口的网络请求 {% sites api:https://api.vlts.cc/output_data/v2/volantis-x/examples %}

### DISCUSSION THREAD

**Discussion by seepine** [volantis-x/hexo-theme-volantis#857] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/93666053/227758393-08ec1b3e-0013-4632-8ee3-06ba2901327b.png)
![image](https://user-images.githubusercontent.com/93666053/227758428-3d19c646-6b8b-4a78-9ae7-121286e043ac.png)

**Comment by seepine** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/93666053/227760438-fddd0b63-0a96-4b7e-b4dd-e21147a4bd71.png)
源码site.js是这么拼接，因此必须有repo才会渲染
```
{%  sites repo:autogen.json  api:https://raw.githubusercontent.com/seepine/blog-friends/output %}
```
且因其中v1是写死的变量，因此提供的数据接口路径必须有包含v1，拼接完是
`https://raw.githubusercontent.com/seepine/blog-friends/output/v1/autogen.json`

**Comment by Tikas** [comment]
总感谢写死 v1 是个问题，由于我的 js 还没开始学，api 实在玩不明白，先留下一个坑，日后学习 js 时，再回来研究
